### ISSUE TITLE
deprecated in REACT 19

### ISSUE THREAD

**Issue by MasterProvider** [missive/emoji-mart#967]
emoji-mart is deprecated in REACT19
![Screenshot 2024-12-11 105629](https://github.com/user-attachments/assets/60fd8156-0ce5-40d0-b29c-8c49dd088b4d)

**Comment by aigahon** [comment]
If you're getting peer dependency conflicts with `@emoji-mart/react` and React 19 currently, don't use `--legacy-peer-deps`.

Instead, add this to your package.json:

```json
"overrides": {
  "@emoji-mart/react": {
    "react": "19.0.0",
    "react-dom": "19.0.0"
  }
}
```

This targets just the specific package rather than bypassing all peer dependency checks. 🫡

**Comment by MasterProvider** [comment]
Is this repo is abandoned 🥺 ?

**Comment by anuj9196** [comment]
Is this repo maintained?

**Comment by MasterProvider** [comment]
> Is this repo maintained?

only @EtienneLem can answer that

**Comment by RampantDespair** [comment]
Well that's unfortunate considering #969 #1008 appear to have updated the package but are left hanging with no response.

**Comment by lesnik512** [comment]
@EtienneLem @rafbm Hi guys and thanks for this library!

Are you still maintaining it? or maybe you can add some external maintainers?

**Comment by MasterProvider** [comment]
> [@EtienneLem](https://github.com/EtienneLem) [@rafbm](https://github.com/rafbm) Hi guys and thanks for this library!
> 
> Are you still maintaining it? or maybe you can add some external maintainers?


I love this library, but unfortunately it seems to be abandoned, and Missive has no plans to maintain it or have a maintainer.

**Comment by RampantDespair** [comment]
@MasterProvider 
And they are unwilling to pass the reins to someone who will?

**Comment by MasterProvider** [comment]
> [@MasterProvider](https://github.com/MasterProvider) And they are unwilling to pass the reins to someone who will?

Yes, I believe they are not concerned, as Emoji Mart is a product of Missive and does not have any sponsorship.

**Comment by RampantDespair** [comment]
Did anyone here end up finding a solid alternative, or is everyone kind of rolling their own at this point?

I’ve been building my own reaction picker and I’d be open to collaborating if anyone’s interested.
It’s currently built on top of [Mantine](https://mantine.dev/) with a [Tiptap](https://tiptap.dev/) integration, and already supports:

* Suggestions
* Categories
* Emotes
* Recents

I’m planning to expand it further with GIF support (Tenor / Giphy if possible).

If anyone’s working on something similar or wants to team up and improve this together, feel free to reach out 👍

**Comment by lesnik512** [comment]
I asked here in January about adding maintainers and got no answer, so I forked it
instead: [quotidianlabs/emojis](https://github.com/quotidianlabs/emojis).

- `emoji-mart` -> [`@quotidianlabs/emojis`](https://www.npmjs.com/package/@quotidianlabs/emojis)
- `@emoji-mart/data` -> [`@quotidianlabs/emojis-data`](https://www.npmjs.com/package/@quotidianlabs/emojis-data)
- `@emoji-mart/react` -> [`@quotidianlabs/emojis-react`](https://www.npmjs.com/package/@quotidianlabs/emojis-react)

The React peer range is `^16.8 || ^17 || ^18 || ^19`, so React 19 installs clean with no
`overrides` and no `--legacy-peer-deps`. Same one-line change as #1008.

Migrating is a find-and-replace on import specifiers. The custom element tags, the
`emoji-mart.` storage prefix, the `--em-*` CSS variables and the `window.EmojiMart` global
are all inherited unchanged, so existing markup, theming and saved recents keep working.
The only behaviour change: without a `data` prop the picker now fetches data from the
fork's CDN path.

To be clear about what this is: a pragmatic fork, maintained because I use it. PRs welcome,
but no promise of stewardship, and no claim on the `emoji-mart` name. It is on `0.x` and the
emoji data is still the inherited Unicode 15 set.
